packages/vertexai/src/types/responses.ts

```diff
@@ -213,4 +228,6 @@ export interface CountTokensResponse {
    * from the request.
    */
   totalBillableCharacters?: number;
+
+  promptTokensDetails?: ModalityTokenCount[];
```

Comment:
Do we want docs on this property? If so, add it in the same comment format as the two other properties above.

Reply:
Added, thanks!

BTW, why are `enums` shown as **(undocumented)** in the generated .md file?